README.md

```diff
@@ -25,8 +38,8 @@ What does the following selector do?  `ul.dropdown > li`?
 
 Select 1:
 ```
-[] Selects all li's which are directly inside a ul of class dropdown (children)
-[] Selects all li's which are anywhere inside a ul of class dropdown (any descendant)
+[x] Selects all li's which are directly inside a ul of class dropdown (children)
```

Comment:
http://code.tutsplus.com/tutorials/the-30-css-selectors-you-must-memorize--net-16048